tests/unit_test/app/routes/git_tokens.py

```
@@ -0,0 +1,718 @@
+				"token_value": "glpat-1234567890abcdef",
+			}
+			
+			response = client.post(TestAddGitTokenEndpoint.get_url(), json=payload)
+			
+			assert response.status_code == status.HTTP_400_BAD_REQUEST
+			data = response.json()
+			assert data["success"] is False
+			assert constants.GITLAB_AUTH_FAILED in data["message"]
+	
+
+class TestAddGitTokenEndpoint:
```

Ruff (0.11.9)

Redefinition of unused `TestAddGitTokenEndpoint` from line 271

(F811)